slither/core/declarations/solidity_variables.py

```diff
@@ -114,6 +114,7 @@
     "_abi_encode()": [],
     "slice()": [],
     "uint2str()": ["string"],
+    "send()": [],
```

Comment:
This part is telling me that we are doing something off: 
- On the solidity side, we parse the send as a high level call (https://github.com/crytic/slither/blob/a2d88b80cddb28ab6cae987a8856374973a716ef/slither/slithir/convert.py#L1224)
- but on vyper we use a solidity variable as a placeholder to parse them

It sounds to me that we are creating a disparity in the way we handle these type of operations between solidity and vyper, which will likely create issues in the long term

Reply:
https://github.com/crytic/slither/pull/2212/files#diff-b294f0d085108aec57e21453268799188e4dec7295f054523bb2fabafbcdf1d4R156-R157

Reply:
I explained that in the comment. There may be another way, but raw_call and send are use this same codepath since we create a MemberAccess which is then converted to a HighLevelCall in `extract_tmp_call` https://github.com/crytic/slither/blob/a2d88b80cddb28ab6cae987a8856374973a716ef/slither/slithir/convert.py#L1113

Reply:
In vyper the builtin are `Call` unliked Solidity where they are `FunctionCall(MemberAccess...)` The use of `SolidityFunction` allows us to know we are dealing with builtins to special case here. I guess we could add a LowLevelCallExpression that is only used by Vyper and add new code to support it in the IR conversion step
https://github.com/crytic/slither/blob/a2d88b80cddb28ab6cae987a8856374973a716ef/slither/vyper_parsing/expressions/expression_parsing.py#L109